swagger.yaml

```diff
@@ -1,14 +1,15 @@
 swagger: "2.0"
+basePath: /v1
 info:
-  title: "dp-nlp-search-scrubber"
+  title: "dp-search-scrubber-api"
   description: "Allows users to identify Output Areas (OA) and Industry Classification (SIC) associated with a given location."
   version: 1.0.0
   license:
     name: "MIT License"
 schemes:
   - http
 paths:
-  /scrubber/search:
+  /search:
```

Comment:
think you missed this - should be /scrubber/search still.

Reply:
I've pushed the changes and merged them our develop, so next time we do a PR they'll be included.